contracts/api3-server-v1/AuctioneerRegistry.sol

```diff
@@ -70,29 +69,30 @@ contract AuctioneerRegistry is Ownable, SelfMulticall {
         // Emit event
     }
 
-    // The OEV proxy metadata is:
+    // The OEV proxy data is:
     //   - Chain ID
     //   - Data feed ID / dAPI name
+    //   - Is dAPI
     //   - Beneficiary address
-    //   - Metadata
+    //   - Data
```

Comment:
I'm referring to the actual/original proxy metadata here so it should remain so

Reply:
Yep, my bad.